.github/workflows/testing.yml

```diff
@@ -93,3 +94,10 @@ jobs:
           interop/interop_test.sh
           cd ${GITHUB_WORKSPACE}/security/advancedtls && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/advancedtls/...
           cd ${GITHUB_WORKSPACE}/security/authorization && go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/authorization/...
+
+      # Reduced set of tests for Go 1.11
+      - name: Run Go1.11 tests
+        if: ${{ matrix.type == 'tests111' }}
+        run: |
+          tests=$(find ${GITHUB_WORKSPACE} -name '*_test.go' | xargs -n1 dirname | sort -r -u | sed "s:^${GITHUB_WORKSPACE}/\?:./:" | sed "s:\/$::" | grep -v ^./security/advancedtls | grep -v ^./credentials/sts | grep -v ^./credentials/tls/certprovider | grep -v ^./credentials/xds | grep -v ^./xds | grep -v ^./security/authorization/engine)
```

Comment:
`sort -r -u`
into
`sort -u` (no need to reverse sort)

`sed "s:^${GITHUB_WORKSPACE}/\?:./:" | sed "s:\/$::"`
into
`sed "s:^${GITHUB_WORKSPACE}:.:"`  (I *think* this works)

`grep -v ^./security/advancedtls | ... | grep -v ^./security/authorization/engine`
into
`grep -v ^./security | ...`

Reply:
Done.